### PR TITLE
Show host graph

### PR DESCRIPTION
**Pull request by liuyu85cn**
please refers to https://github.com/vesoft-inc/nebula-graph/pull/437

related pr:

graph: https://github.com/vesoft-inc/nebula-graph/pull/437
common: https://github.com/vesoft-inc/nebula-common/pull/325
storage: https://github.com/vesoft-inc/nebula-storage/pull/223

### CODE REVIEW

File: src/kvstore/NebulaStore.h [hunk 1]
```diff
@@ -264,6 +264,10 @@ class NebulaStore : public KVStore, public Handler {
                               PartitionID partId,
                               const std::vector<HostAddr>& remoteListeners) override;
 
+    raftex::RaftexService* getRaftexService() const {
```
Comment (critical27) [inline src/kvstore/NebulaStore.h:267]: There is an method `part` in `NebulaStore`, you could just use it.

Comment (liuyu85cn) [reply src/kvstore/NebulaStore.h]: good catch, fixed

**Review by panda-sheep — APPROVED** [review]
Good job！

**Review by critical27 — DISMISSED** [review]
Good job

**Review by panda-sheep — DISMISSED** [review]

**Review by CPWstatic — APPROVED** [review]

**Review by critical27 — APPROVED** [review]
LGTM

**Pull request merged**